resources/local/config.toml

```diff
@@ -242,6 +242,16 @@ tarpit_chance = 0.2
 # How long peers remain blocked after they get blocklisted.
 blocklist_retain_duration = '1 minute'
 
+# Used to control if a timed-out request should make the consecutive reqeusts to fail.
+# It is recommended to set this to `true` unless network connectivity issues are being troubleshot.
+bubble_timeouts = true
+
+# The maximum time a peer is allowed to take to receive an error.
```

Comment:
I recommend changing this "a fatal error", since we always close the connection after receiving one.

Reply:
Updated in 6a891781a543224cfcc8c5f875575bf06cd00c77